ui/src/app/components/webcomponents/select-test-lab.component.ts

```diff
@@ -127,16 +127,16 @@ export class SelectTestLabComponent implements OnInit {
   ngOnInit(): void {
     this.integrationsService.findAll().subscribe(res => {
       this.applications = res;
-      let labType = TestPlanLabType.Hybrid
       if(this.isNewTestPlan) {
+        let labType = TestPlanLabType.Hybrid
         if(this.executionEnvironment){
           labType = this.executionEnvironment.testPlanLabType
         }
         else if (this.isTestsigmaLabInstalled) {
           labType = TestPlanLabType.TestsigmaLab
         }
+        this.selectTestLabForm.controls[TestLabFormControls.TESTPLAN_LABTYPE].setValue(labType)
       }
-      this.selectTestLabForm.controls[TestLabFormControls.TESTPLAN_LABTYPE].setValue(labType)
```

Comment:
@manikanta-reddy-t What should be the lab type if this.isNewTestPlan is false ? Have you verified for the other case ?

Reply:
By default it opens testsigmalab if it is false